### PR TITLE
Upgrade demo version

### PR DESCRIPTION
**Pull request by Oneill38**
Upgrading Ruby, Rails, and the LD gem to more recent versions to keep the example app current. No additional changes were made other than doing version updates. Updating the rails version did require a migration, I added that step to the README

### CODE REVIEW

**Comment by Oneill38** [comment]
I'm unable to add any reviewers but @bwoskow-ld it looks like you've contributed before, would you be able to give this a review?

File: config/environments/development.rb [hunk 1]
```diff
@@ -12,22 +14,26 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  # Enable server timing
+  config.server_timing = true
+
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join('tmp', 'caching-dev.txt').exist?
+  if Rails.root.join("tmp/caching-dev.txt").exist?
```
Comment (keelerm84) [inline config/environments/development.rb:22]: Why this change?

**Comment by keelerm84** [comment]
Thank you again for all of your work on this!
